service/controller/resource/oauth/test/case-0-cluster-api.golden

```diff
@@ -0,0 +1,25 @@
+apiVersion: v1beta1
+kind: Ingress
+metadata:
+  creationTimestamp: null
+  labels:
+    app.kubernetes.io/instance: bob
+    app.kubernetes.io/managed-by: prometheus-meta-operator
+    app.kubernetes.io/name: prometheus
+  name: bob
+  namespace: bob-prometheus
+spec:
+  rules:
+  - host: https://prometheus
+    http:
+      paths:
+      - backend:
+          serviceName: oauth2-proxy
```

Comment:
This won't work since the `oauth2-proxy` is running in the `monitoring` namespace.